src/components/RectangularButton.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { StyleSheet, View, Text, Pressable } from "react-native";
+
+export default function RectangularButton({ text }: { text: string }) {
```

Comment:
In general we'll be using the function declaration of this form:

const RectangularButton = () => {
}

export default RectangularButton;

This works totally fine, but let's keep it this way for the sake of consistency.
You can check all the existing screens for a reference on this format!